ovos_tts_plugin_piper/espeak_wrapper.py

```
@@ -0,0 +1,378 @@
+
+        Returns:
+            str: The validated language code.
+
+        Raises:
+            ValueError: If the language code is unsupported.
+        """
+        if target_lang in valid_langs:
+            return target_lang
+        best_lang = "und"
+        best_distance = 10000000
+        for l in valid_langs:
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)